src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1123,7 +1123,7 @@ private String execDockerCmd(String command, int timeout) {
             // the timeout was too short and the docker command has not yet completed. There is no exit value.
             debug("IllegalThreadStateException, message="+e.getMessage());
             error("The docker command did not complete within the timeout period: " + timeout + " seconds. " +
-                "Use the dockerTimeout option to specify a longer period or investigate issues running docker. ", e);
+                "Use the dockerTimeout option to specify a longer period or investigate performance issues with running the docker command. ", e);
```

Comment:
Now the dockerTimeout option is not relevant to this message if the executed command was docker ps or docker stop.

Reply:
Displaying a message specific to docker build....